src/utils/helpers.ts

```diff
@@ -142,3 +143,26 @@ export const abbreviateNumber = (value: number): string =>
   numeral(value).format('0.[00]a').toUpperCase();
 
 export const getShadow = () => 'box-shadow: 0px 4px 12px rgba(0, 0, 0, 0.07);';
+
+const getBreakpoint = (i, theme) =>
+  `@media screen and (min-width: ${theme.breakpoints[i]})`;
+
+export const parseResponsiveStyles =
```

Comment:
How much effort would it be to enable the object notation?

Reply:
not sure what you mean, could you expand the description?